web/src/lib/utils.ts

```diff
@@ -0,0 +1,30 @@
+import {
+	type Chain,
+	type ChainID,
+	type FungibleToken,
+	type TokenSymbol
+} from '@chainsafe/gopher-sdk';
```

Comment:
I know web is just a playground for the SDK, but I'll leave this comment haha

if you do the type import like this:

```suggestion
import type {
	Chain,
	ChainID,
	FungibleToken,
	TokenSymbol
} from '@chainsafe/gopher-sdk';
```

the import will be completely removed after build. if you leave it like this, after build you'll still have `import {} from '@chainsafe/gopher-sdk';`

Reply:
Sveltes linter did that way :man_shrugging: